src/deck-options.md

```diff
@@ -474,6 +480,18 @@ If you're first switching from SM-2 and still wish to use this option, we recomm
 
 Compute minimum recommended retention (CMRR) attempts to find the desired retention value that leads to the most material learnt, in the least amount of time. The calculated number can serve as a reference when deciding what to set your desired retention to. You may wish to choose a higher desired retention, if you’re willing to trade more study time for a greater retention rate. However, setting your desired retention lower than the minimum is not recommended, as you'll spend more time studying than necessary, due to increased forgetting.
 
+### The Simulator
+
+In the latest version of Anki, you can use the simulator to get an estimate of your workload,
+either in reviews per day or in minutes of studying per day.
+**Days to simulate** controls the duration of the simulated study history. **Additional new cards to simulate** 
```

Comment:
These all run together in a paragraph, and would be better separated into list elements like some other sections of this page.